config/locales/user_mailer/en.yml

```diff
@@ -63,17 +63,18 @@ en:
         email addresses. We recommend that you also change your password.
       subject: New email address added
     email_confirmation_instructions:
+      create_new_account: create a new account.
       first_sentence:
         confirmed: Trying to change your email address?
-        forgot_password: You tried to reset your login.gov password but we don't have
-          an account linked to this  email address. If you'd like to set up a new
-          account linked to this email, confirm your email address below.
+        forgot_password_html: "<p>You tried to reset your login.gov password but we
```

Comment:
I think it would be easier to break this in to two keys and put the `<p>` tags in the template? In my experience, leaving HTML inside of translations often get messed up and broken in other locales